### DISCUSSION TITLE
While resolving module `zustand/middleware/immer`, the Haste package `zustand` was found. However the module `middleware/immer` could not be found within the package

### DISCUSSION THREAD

**Discussion by ManuelJrez** [pmndrs/zustand#1595] [Q&A]
i'm trying to use the immer middleware using the latest version of zustand but i get the error of the title 😓

**Comment by dai-shi** [comment]
Can you create a minimal reproduction?

**Reply by ManuelJrez** [reply]
Yes, i can but i tested yesterday using react and it's working fine, my issue is trying to use with react native 😓 @dai-shi

**Reply by dai-shi** [reply]
I see... Does it work with other middleware? Like `combine`?

**Reply by ManuelJrez** [reply]
it's weird because now i have this error `undefined is not a function` with the basic example of the readme 😓 

import {create} from 'zustand';

const useBearStore = create(set => ({
  bears: 0,
  increasePopulation: () => set(state => ({bears: state.bears + 1})),
  removeAllBears: () => set({bears: 0}),
}));

**Reply by dai-shi** [reply]
are you using the latest version of `zustand`?

**Reply by ManuelJrez** [reply]
yes, i use this version `"zustand": "^4.3.2"`

**Reply by ManuelJrez** [reply]
@dai-shi the error is using the latest version of react native `0.72.0`, i tried the latest version of zustand using an older RN version (`0.68.0`) and it's working fine (both using typescript)

**Reply by dai-shi** [reply]
Good finding! We need someone who is familiar with RN to help here.
